doc/build/orm/loading_columns.rst

```diff
@@ -114,11 +120,12 @@ option (described later in this section) to defer all ``Book`` columns except
 those explicitly specified::
 
     from sqlalchemy.orm import joinedload
+    from sqlalchemy import select
 
-    query = session.query(Author)
-    query = query.options(
-                joinedload(Author.books).load_only(Book.summary, Book.excerpt),
-            )
+    stmt = select(Author)
+    stmt = stmt.options(
+        joinedload(Author.books).load_only(Book.summary, Book.excerpt)
+    )
```

Comment:
I see here I'm not executing....that's fine lets just change the examples 1-1 for now

Reply:
Yes, there are a bunch of those cases in this file. I assumed the discrepancy with the other examples wasn't intentional, so I thought it'd be best to adhere to one style. Anyway, I've reverted those changes and they should be 1-1 translations of the 1.x style examples now. 

Maybe adding more consistency here would be best addressed separately.